### PR TITLE
Fix README.md for 2.0

### LINKED ISSUE #2434: default docs example for 2.0 is not working
I copy pasted this code: 

```
import { ApolloClient, HttpLink } from 'apollo-client-preset';

const client = new ApolloClient({
  link: new HttpLink({
    uri: 'https://graphql.example.com',
  }),
});
```

and this is not working: 
Error: 
        In order to initialize Apollo Client, you must specify link & cache properties on the config object.
        This is part of the required upgrade when migrating from Apollo Client 1.0 to Apollo Client 2.0.
        For more information, please visit:
          https://apollographql.com/docs/react/setup
        to help you get started.

### PR DESCRIPTION
**Pull request by misoguy**
Fix #2434

### CODE REVIEW

**Comment by jbaxleyiii** [comment]
Deploy preview ready!

Built with commit 47df86a0fcc111643db240b9a836ab43772b7fb3

https://deploy-preview-2465--apollo-client-docs.netlify.com

File: README.md [hunk 1]
```diff
@@ -43,12 +46,12 @@ const client = new ApolloClient();
 To point `ApolloClient` at a different URL, just create your own `HttpLink` instance, like so, replacing `https://graphql.example.com` with your GraphQL API's URL:
 
 ```js
-import { ApolloClient, HttpLink } from 'apollo-client-preset';
```
Comment (peggyrayzis) [inline README.md]: You can also import InMemoryCache from `apollo-client-preset`.

**Comment by peggyrayzis** [comment]
Thanks @misoguy for your contribution! This looks great 🎉 

If you spot any other issues with the docs, please feel free to make a PR. 😀

**Comment by misoguy** [comment]
@peggyrayzis Thank you for merging this PR 😄

**Pull request merged**